DeRhamCohomology/DifferentialForm.lean

```diff
@@ -7,12 +7,68 @@ noncomputable section
 open Filter ContinuousAlternatingMap
 open scoped Topology
 
-variable {E F : Type*} [NormedAddCommGroup E] [NormedSpace ℝ E]
-  [NormedAddCommGroup F] [NormedSpace ℝ F] {n : ℕ}
+variable {E F G : Type*} [NormedAddCommGroup E] [NormedSpace ℝ E]
+  [NormedAddCommGroup F] [NormedSpace ℝ F]
+  [NormedAddCommGroup G] [NormedSpace ℝ G] {n k : ℕ}
 
 -- TODO: change notation
 notation "Ω^" n "⟮" E ", " F "⟯" => E → E [⋀^Fin n]→L[ℝ] F
 
+variable {v : E}
+variable (ω τ : Ω^n⟮E, F⟯)
+variable (f : E → F)
+
+instance : Add (Ω^n⟮E, F⟯) :=
```

Comment:
Since `Ω^n` is a `notation, Lean should be able to synth `Add` and other instances.